### ISSUE TITLE
[BUG] Grid region resizing is not accessible

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#13432]
Related to https://github.com/bryntum/support/issues/11914

A Grid must only contain roles `rowgroup` and/or `row`. And a `rowgroup` must only contain role `row` etc.

A subGrid (the resizable child in a grid) itself contains up to three splitter elements (This are *not* Splitter widgets, they are just extra elements from which we listen for delegated mouse drag gestures). One in its header, one in its body and one in its footer. Making them accessible with a role would be invalid. And there is no standard way of focusing a splitter. Arrow keys move from cell to cell as per the ARIA standard. TAB tabs out of the grid to the following (or previous if using SHIFT) sibling.

Could be that we implement a `keyMap` key for expanding, shrinking or collapsing the subGrid in which we are focused. But then how to expand a collapsed subGrid? - It's gone and no longer focusable. The following sibling would have to be able to expand its previous sibling.

Maybe a single keystroke could pop up a whole modal dialog which describes the subGrid layout in normal HTML inputs for widths and checkboxes for collapsed etc which could then be manipulated normally to control the subGrid layout under the dialog.

To be discussed with @matsbryntse @isglass and Aleksei from Certinia.

**Comment by ExtAnimal** [comment]
Alternatively the column menu gets a new entry for each subGrid added by the RegionResize Feature.

Each entry has a submenu with a checkbox for collapsed state and an input field for the width.

```
Locked -> Collapsed [x]
          Width : n
```
